docs/self-service-account-api-descriptor.yml

```diff
@@ -38,11 +38,32 @@ paths:
       tags:
         - Self Account
       summary: Initiates dialog to change user password
+      consumes:
+        - application/x-www-form-urlencoded
+      produces:
+        - application/json
+      parameters:
+        - in: body
+          name: password
+          description: The naw password value.
```

Comment:
Should be in the post method and we should consumes application/json no and produces nothing WDYT ?